packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/lib/getDrupalSearchResults.js

```diff
@@ -0,0 +1,23 @@
+import { defaultFetch } from '@pantheon-systems/drupal-kit';
+import { ServerResponse } from 'http';
+
+import { NextResponse } from 'next/server';
+
+export const getDrupalSearchResults = async (id, index) => {
```

Comment:
I would expect this to live in `drupal-kit` then imported to this add-on and implemented. That way the `getDrupalSearchResults` could be framework agnostic and be potentially reused later.

Reply:
Thanks for breaking apart the different pieces here. With everything you mentioned above, I am also lean towards keeping it where it is with being open to refactoring as the project develops.